cadwyn/structure/versions.py

```diff
@@ -448,7 +448,10 @@ async def _convert_endpoint_response_to_version(
             path,
             method,
         )
-        if isinstance(response_or_response_body, FastapiResponse):
+        if isinstance(response_or_response_body, FastapiResponse) and response_info.body:
+            # None is a valid response body, when a user returns just a status code
+            # dumping it will result in a 'null' string, which doesnt correlate with the content-length
```

Comment:
What do you mean by "it doesn't correlate wth content length"?

Reply:
uvicorn calculates the body at the endpoint level. In this case its None there, meaning `content-length=0`
`json.dums(None)` results into `null`, and `content-length` should be 4, but it was already calculated to 4

Reply:
Okay. Let's copy-paste this exact explanation into code comments for more clarity.

Reply:
added